ext_tables.sql

```diff
@@ -277,42 +234,15 @@ CREATE TABLE tx_cart_domain_model_coupon (
     number_used int(11) DEFAULT '0' NOT NULL,
 
     INDEX `parent` (pid), INDEX `t3ver_oid` (t3ver_oid,t3ver_wsid),
-    PRIMARY KEY (uid)
-);
-
-#
-# Table structure for table 'tx_cart_domain_model_cart'
-#
-CREATE TABLE tx_cart_domain_model_cart (
```

Comment:
We shouldn't remove this table because its required by several payment provider extensions (have to be upgraded).

Reply:
Okay so that is what I was missing. I was really wondering how you missed removing that over the years :D